user_docs/en/changes.t2t

```diff
@@ -7,6 +7,7 @@ What's New in NVDA
 
 == New Features ==
 - In Windows 11, NVDA will announce alerts from voice typing and suggested actions including the top suggestion when copying data such as phone numbers to the clipboard (Windows 11 2022 Update and later). (#16009, @josephsl)
+- Added same style and different style navigation QuickNav commands, not assigned to any gestures.

```

Comment:
1. quick nav -> quick navigation
2. Add the issue number
3. Added credit to you; I think that you have added it in the other PRs, so I guess you agree also for this one.

```suggestion
- Added same style and different style quick navigation commands, not assigned to any gestures. (#16000, @mltony)
```